docs/source/helper.py

```
@@ -310,6 +310,15 @@ def generate_llmapi():
     public_classes_names = extract_all_and_eval(llmapi_all_file)['__all__']
 
     content = underline("API Reference", "-") + "\n\n"
+    content += ".. note::\n"
+    content += "    :collapsible:\n"
+    content += "    Since version 1.0, we attach a status label to every API endpoint and every parameter.\n\n"
```

Ruff (0.12.2)

String contains ambiguous ` ` (NARROW NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)